### ISSUE TITLE
how can I upload my local image set and create a new dataset in another windows or ubuntu machine?

### ISSUE THREAD

**Issue by JulyUp** [NVIDIA/DIGITS#1212]
When I do ./digits-devserverand use 'Use Text FIles' in module 'New Image Classification Dataset', how can I upload my local image set and create a new dataset in another windows or ubuntu machine? I tried and failed. It seems digits can not find the image.
Thank you.

**Comment by lukeyeager** [comment]
That's right - those files need to be on the same server where DIGITS is running. Sorry for the inconvenience.

**Comment by JulyUp** [comment]
Looking forward to adding this function.

**Comment by gheinrich** [comment]
The files need to be accessible from the DIGITS server but they can be hosted on a different machine if they are mapped on an NFS volume, for example.

**Comment by lukeyeager** [comment]
Renamed and recategorized as a feature request

**Comment by JulyUp** [comment]
@lukeyeager @gheinrich  hello,when will you add this function? Now I need to upload my dataset and train in other machine.

**Comment by gheinrich** [comment]
We have no plans to do this in the foreseeable future. As I suggested on https://github.com/NVIDIA/DIGITS/issues/1212#issuecomment-256321479 you might want to use standard networking tools to make the files accessible from the DIGITS server.

**Comment by JulyUp** [comment]
@gheinrich I have done this function though upload a zip file then upzip it.
